torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -406,6 +738,32 @@ def aot_dispatch_autograd(
                 + num_tokens  # See Note [Side-Effectful Tokens in AOTAutograd]
             )
             fake_mode = detect_fake_mode()
+            # TODO(anijain2305) - this changes fx_g signature. But somehow it still
+            # works. I don't understand why it works. There will be some place where we
+            # let go of fx_g and use fw_module and bw_module. The question is - does
+            # joint graph partitioners need to keep the invariant of original fx_g
+            # signature?
+            fx_g = run_joint_graph_passes_on_hops(fx_g, joint_inputs, aot_config)
+            if aot_config.enable_log:
+                aot_joint_log.info(
+                    "%s",
+                    lazy_format_graph_code(
+                        "Joint graph",
```

Comment:
One kind of annoying thing over time is that `TORCH_LOGS="aot"` has become increasingly noisy: Previously in the training case it used to log:

(1) the joint graph
(2) the fw graph
(3) the bw graph

And now it will log:

(1) joint graph (before invoke_hop)
(2) joint graph (after invoke_hop graph pass)
(3) forward graph
(4) backward graph

Reply:
A few ideas:

(1) can you change the name to make it clear what distinguishes the two joint graphs in the logs? e.g. `Joint graph (post invoke_subgraph graph pass)`)

(2) One thing that would be nice is: in most cases, we don't see any `invoke_subgraph` calls in the overall graph at all, and it feels redundant/spammy to always log two graphs that are identical. How hard would it be to tweak `run_joint_graph_passes_on_hops()` so that it can tell when it is a big no-op, and only log the new graph when it actually found an `invoke_subgraph` call in the outer graph?

Reply:
This was left over from debugging. I don't need it in this PR. I can address in a separate PR. Removing.